### ISSUE TITLE
Please make option to use Babel instead

### ISSUE THREAD

**Issue by kristianmandrup** [briandipalma/slush-node-esnext#1]

**Comment by briandipalma** [comment]
HI @kristianmandrup this project isn't maintained. I don't think slush is still alive either.

**Comment by kristianmandrup** [comment]
Slush is dead!? Then what is the best generator these days?

**Comment by briandipalma** [comment]
Well it might not be dead but it doesn't seem to be moving too much.

Compare:

https://slushjs.github.io/generators/#/ with http://yeoman.io/generators/ and
https://github.com/slushjs with https://github.com/yeoman

I would use either a good GitHub repository or a tool like `react-create-app` as a project starting point over a generator.

**Comment by kristianmandrup** [comment]
Yeah, I noticed the new trend seems to be to create "generators" from scratch, using `inquirer` and a template library of choice, and then publish it with a custom binary like `react-create-app`.
It's a pity - I feel tooling and good libs is the most important for productive/fun development :)
